autorun.brs

```diff
@@ -239,13 +230,11 @@ Sub CreateHtmlWidget()
     mouse_enabled: true,
     javascript_enabled: true,
     brightsign_js_objects_enabled: true,
+    nodejs_enabled: true,
     storage_path: "./fugo-storage",
+    storage_quota: 1073741824,
     security_params: {
       websecurity: false,
-    },
-    inspector_server: {
```

Comment:
I have a suspicion that this makes player to freeze. Previously, we tried fixing it by [refreshing the PlayerJS page](https://github.com/OutOfAxis/fugo-player/commit/c95be8a9d0da644191919b84daf596c1a825ed58#diff-0b5adbfe7b36e4ae2f479291e20152e33e940f7f265162d77f40f6bdb5da7405R42) but I'm pretty sure it didn't help.